nucypher-core/Cargo.toml

```diff
@@ -14,8 +14,16 @@ categories = ["cryptography", "no-std"]
 umbral-pre = { git = "https://github.com/piotr-roslaniec/rust-umbral.git", rev = "90a5075bb3462421fa6db79463960cd8e10a7025", features = ["serde"] }
 ferveo = { package = "ferveo-pre-release", version = "0.1.0-alpha.7" }
 serde = { version = "1", default-features = false, features = ["derive"] }
-generic-array = "0.14"
+generic-array = { version="0.14", features = ["zeroize"] }
 sha3 = "0.10"
 rmp-serde = "1"
 serde_with = "1.14"
 hex = "0.4"
+hkdf = "0.12.3"
+sha2 = "0.10.6"
+x25519-dalek = { version="2.0.0-rc.2", features = ["serde", "static_secrets"] }
+chacha20poly1305 = "0.10.1"
+zeroize = { version="1.6.0", features = ["derive"] }
+rand_core = "0.6.4"
+rand_chacha = "0.3.1"
+rand = "0.8.5"
```

Comment:
Just curious, do you guys not have an "ensure newline at eof on save" setting in the editors? I noticed Piotr does that too :)